app/uk/gov/hmrc/servicecommissioningstatus/service/StatusCheckService.scala

```diff
@@ -38,6 +38,9 @@ class StatusCheckService @Inject()(
 )(implicit ec: ExecutionContext){
 
   import scala.jdk.CollectionConverters._
+
+  private val CREATE_A_REPO = "https://build.tax.service.gov.uk/job/PlatOps/job/Tools/job/create-a-repository/build"
```

Comment:
Java constant naming convention?  `CreateARepo` would be more idiomatic, but since all vals are constant `createARepo` is more common.
Since it's only used once, it could be inlined again like the others?

Reply:
removed it as uneeded